### PR TITLE
Fix bank error notif

### PR DESCRIPTION
**Pull request by rohitpaulk**
#3591

### CODE REVIEW

File: gratipay/utils/notifications.py [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
         ['a',
-            {'href': href}, _("The transfer to your bank account has failed!"),
+            {'href': href}, _("Your last paypal payout failed!"),
         ],
     )
 
 
 def credit_card_failed(_, user):
-    href = '/%s/routes/credit-card.html' % user.participant.username
+    href = '~/%s/routes/credit-card.html' % user.participant.username
```
Comment (whit537) [inline gratipay/utils/notifications.py]: These should start with `/~` instead of just `~`, right?

File: gratipay/utils/notifications.py [hunk 2]
```diff
@@ -1,16 +1,14 @@
-
-
-def ba_withdrawal_failed(_, user):
-    href = '/%s/routes/bank-account.html' % user.participant.username
+def paypal_withdrawal_failed(_, user):
+    href = '~/%s/routes/paypal.html' % user.participant.username
     return ('error',
         ['a',
-            {'href': href}, _("The transfer to your bank account has failed!"),
+            {'href': href}, _("Your last paypal payout failed!"),
```
Comment (whit537) [inline gratipay/utils/notifications.py]: Let's observe the preferred casing for `PayPal`.

**Comment by rohitpaulk** [comment]
FIxed, @whit537.

File: sql/branch.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+BEGIN;
+    UPDATE participants
+       SET notifications = array_remove(notifications, 'ba_withdrawal_failed')
+     WHERE 'ba_withdrawal_failed' = ANY(notifications);
```
Comment (whit537) [inline sql/branch.sql]: This is new SQL for me. `ANY` , eh?

Comment (whit537) [inline sql/branch.sql]: http://www.postgresql.org/docs/9.3/static/functions-comparisons.html

Comment (whit537) [inline sql/branch.sql]: !m @rohitpaulk

**Pull request merged**